README.md

```diff
@@ -587,6 +587,7 @@ Table of Contents
   * [netlify.com](https://www.netlify.com/) — Builds, deploy and hosts static site or app, free for 100 GB data and 100 GB/month bandwidth
   * [pages.github.com](https://pages.github.com/) — Hosting static site directly from GitHub repository
   * [pantheon.io](https://pantheon.io/) — Drupal and WordPress hosting, automated DevOps and scalable infrastructure. Free for developers and agencies
+  * [pythonanywhere.com](https://www.pythonanywhere.com/) — Python hosting,develop and host your website or any other code directly from your browser without having to install software. 
```

Comment:
is already under PaaS :) maybe thats a bad category for it?

Reply:
I guess hosting would be better.

Reply:
feel free to move it then :)